### DISCUSSION TITLE
Playground for this plugin?

### DISCUSSION THREAD

**Discussion by mandrasch** [zackad/prettier-plugin-twig#29] [Ideas]
Hey, awesome that there is an active fork of the twig-melody plugin! 🤩  Thanks for creating it! 👏  (The Craft CMS community would really love a maintained prettier plugin I guess, especially if some craftcms-flavored twig would also be supported https://github.com/trivago/prettier-plugin-twig-melody/issues/47)

Just a quick idea - a playground would maybe be cool to test the plugin.  I saw https://github.com/loilo/prettier-php-playground/tree/master, maybe this could just be forked for this plugin.

I'll check in the next days if I have time if this is easily achievable. Just wanted to post it here as idea in case someone is faster than me. 

Cheers!

**Comment by rellafella** [comment]
This sounds like a great idea @mandrasch!

**Comment by zackad** [comment]
For now I've forked https://github.com/loilo/prettier-php-playground and published at https://zackad.github.io/prettier-twig-playground. Don't have time to actually implementing it yet. If someone want to give it a try, please go ahead.

playground: https://zackad.github.io/prettier-twig-playground
repo: https://github.com/zackad/prettier-twig-playground
